stdlib/auth/src/main/ballerina/src/auth/utils.bal

```diff
@@ -42,13 +42,17 @@ public const string AUTH_SCHEME_BASIC = "Basic ";
 # Prefix used to denote Bearer Authentication scheme.
 public const string AUTH_SCHEME_BEARER = "Bearer ";
 
-# The table name of the config user section of the TOML file.
+# The table name specified in the user section of the TOML configuration.
 const string CONFIG_USER_SECTION = "b7a.users";
 
-# Extracts the username and password from the credential values.
+# Extracts the username and the password from the credential values.
```

Comment:
```suggestion
# Extracts the username and the password from the base64 encoded `username:password` value.
```

Reply:
# Extracts the username and the password from the base64-encoded `username:password` value.